handler/core/src/main/java/io/knotx/fragments/handler/action/InMemoryCacheActionFactory.java

```diff
@@ -67,24 +70,26 @@ public String getName() {
   }
 
   @Override
-  public Action create(String alias, JsonObject config, Vertx vertx, Action doAction) {
-
+  public Action create(String alias, JsonObject config, Vertx vertx, Action doAction){
     return new Action() {
+      ActionLogger actionLogger = ActionLogger.create(alias, config);
       private Cache<String, Object> cache = createCache(config);
       private String payloadKey = getPayloadKey(config);
 
       @Override
       public void apply(FragmentContext fragmentContext,
           Handler<AsyncResult<FragmentResult>> resultHandler) {
-
         String cacheKey = getCacheKey(config, fragmentContext.getClientRequest());
         Object cachedValue = cache.getIfPresent(cacheKey);
+        actionLogger.info("cached_key", cacheKey);
```

Comment:
```suggestion
        actionLogger.info("key", cacheKey);
```
still not `cached_key`, more `cache_key`.